### PR TITLE
feat: split contact from member creation

### PR DESCRIPTION
**Pull request by Besan-Darawsheh**
https://wix.monday.com/boards/9601070284/pulses/10905996464

### CODE REVIEW

File: backend/members-area-methods.js [hunk 1]
```diff
@@ -2,10 +2,11 @@ const { auth } = require('@wix/essentials');
 const { members, authentication } = require('@wix/members');
 const elevatedCreateMember = auth.elevate(members.createMember);
 
-function prepareContactData(partner) {
+function prepareMemberData(partner) {
   const phones = Array.isArray(partner.phones) ? partner.phones : []; //some users don't have phones
   const options = {
     member: {
+      //Keeping contact creation in member data for future purposes, in case we need to use it later
```
Comment (Besan-Darawsheh) [inline backend/members-area-methods.js:9]: still need to revisit & test this to see if it would be better just to remove it so we won't get duplicate info

File: backend/members-data-methods.js [hunk 2]
```diff
@@ -42,10 +42,14 @@ async function createContactAndMemberIfNew(memberData) {
       phones: memberData.phones,
       contactFormEmail: memberData.contactFormEmail || memberData.email,
     };
-    const contactId = await createSiteMember(toCreateMemberData);
+    const [wixMemberId, wixContactId] = await Promise.all([
+      createSiteMember(toCreateMemberData),
+      createContact(toCreateMemberData),
```
Comment (matheuswix) [inline backend/members-data-methods.js:47]: @Besan-Darawsheh maybe here should be createSiteContact. Just to be align with the other function

**Review by matheuswix — APPROVED** [review]

File: backend/members-data-methods.js (hunk 2 above)
Comment (Besan-Darawsheh) [reply backend/members-data-methods.js:47]: changed in next PR [here](https://github.com/psdevteamenterprise/abmp-npm/pull/51/files#diff-c386317c01071471465eedff6691b23fdf4557758756b3a1e3367d11899e7ebdR14)
